app/controllers/refinery/authentication/devise/admin/users_controller.rb

```diff
@@ -63,7 +63,7 @@ def create_successful
             # if the user is a superuser and can assign roles according to this site's
             # settings then the roles are set with the POST data.
             if user_can_assign_roles?
-              @user.roles = @selected_role_names.map { |r| Refinery::Authentication::Devise::Role[r.downcase] }
+              @user.roles = @selected_role_names.map(&:underscore)
```

Comment:
this was supposed to turn the array into roles

```suggestion
              @user.roles = @selected_role_names.map { |r| Refinery::Authentication::Devise::Role[r.underscore] }
```